base/ca/src/com/netscape/cms/servlet/cert/scep/CRSEnrollment.java

```diff
@@ -404,6 +414,31 @@ public void service(HttpServletRequest httpReq,
 
     }
 
+    /*
+     * If this is a POST request, try reading the binary message from the request body.
+     */
+    private String readMessageFromPostBody(HttpServletRequest httpReq) {
+        String message = null;
+
+        try {
+            if (httpReq.getMethod().equalsIgnoreCase("POST")) {
```

Comment:
just a style thing.  Personally, I'd put the the check on the method (POST or GET) outside of this method and branch on how each method retrieves its message.  But I'll let our our refactor expert @edewata chip in on this.

Reply:
I'm not too familiar with SCEP. Does it support both POST and GET? In general if POST and GET are handled differently I'd suggest putting them into separate `doPost()` and `doGet()` methods.